dockers/docker-base/etc/rsyslog.d/supervisor.conf

```diff
@@ -1,9 +1,9 @@
-$ModLoad imfile
-
-$InputFileName /var/log/supervisor/supervisord.log
-$InputFileTag supervisord
-$InputFileStateFile state-supervisor
-$InputFileSeverity info
-$InputFileFacility local0
-$InputFilePersistStateInterval 1
-$InputRunFileMonitor
+module(load="imfile") #By default notify mode
```

Comment:
could you please explain why this file format/style is changed? And why did the previous specify polling mode? Not seeing it in the config. It is default in the old format as well? Thanks!

Reply:
@hui-ma yes old semantics of rsyslogd default to polling mode. Rsyslogd document recommends to use notify mode and new semantic going forward

Reply:
Thanks!

Reply:
Sorry for the late reply due to the holidays, but I wonder if we should explicitly set `mode="inotify"` just in case the default mode were ever to change.

Reply:
Thanks @jleveque . PR https://github.com/Azure/sonic-buildimage/pull/6351 has the change.